storage/src/global_state/error.rs

```diff
@@ -33,6 +33,10 @@ pub enum Error {
     /// Failed to put a trie node into global state because some of its children were missing.
     #[error("Failed to put a trie into global state because some of its children were missing")]
     MissingTrieNodeChildren(Digest, TrieRaw, Vec<Digest>),
+
+    /// Failed to prune listed keys.
+    #[error("Pruning attempt failed.")]
+    FailedToPrune(Vec<Key>),
```

Comment:
It seems like we don't use the un-pruned keys. Is there any reason to pass them on in the error? Do we want to print them?

Reply:
the logic i moved this from was doing so...leaving it thus for now. we can certainly do a optimization pass after it all shifted.